rw-system.sh

```diff
@@ -203,6 +203,7 @@ if getprop ro.vendor.build.fingerprint | grep -q -i -e xiaomi/wayne -e xiaomi/ja
     setprop persist.imx376_sunny.light.lux 280
     setprop persist.imx376_ofilm.low.lux 310
     setprop persist.imx376_ofilm.light.lux 280
+    setprop ro.sf.lcd_density 460
```

Comment:
It is not defined by vendor?
I haven't ever seen anyone complaining about this..?

Reply:
Nah basically on the Mi A2 / Mi 6X, the Quick Settings toggles are like right up against the sides of the panel. It's not right. I'll include a picture of before and after this change:

https://photos.app.goo.gl/kJasoSBxwSXzF22n6